panoptes_client/subject.py

```diff
@@ -215,8 +215,6 @@ def set_raw(self, raw, etag=None, loaded=True):
         super(Subject, self).set_raw(raw, etag, loaded)
         if loaded and self.metadata:
             self._original_metadata = deepcopy(self.metadata)
-        elif loaded:
```

Comment:
OOC, what was the motivation for this change on Subject.py? Is it related to this change?

Reply:
I used the same pattern related to `metadata` and `_original_metadata` in the SubjectSet class as is used for the Subject class. Therefore I applied the same fix to Subjects as was used for SubjectSets. We realized that the "how to initialize with empty metadata" issue affected both classes, but hasn't been reported as a bug for Subjects due to the fact that few subjects are created without metadata.